api.md

```diff
@@ -1354,6 +1584,72 @@ enterprise | f       | 2019-02-15 13:44:53-05
 (1 row)
 ```
 
+---
+## timescaledb_information.policy_stats[](timescaledb_information-policy_stats)
+
+Show information and statistics about policies created to manage data retention and other administrative tasks on hypertables. (See policies)
+The statistics include information about when and whether the background job used to implement the policy succeeded and when it is scheduled to run next etc. 
+
+#### Available Columns
+
+|Name|Description|
+|---|---|
+| `hypertable` | The name of the hypertable on which the policy is applied |
+| `job_id` | The id of the background job created to implement the policy |
+| `job_type` | The type of policy the job was created to implement |
+| `last_run_success` | Whether the last run succeeded or failed |
+| `last_finish` | The time the last run finished|
+| `last_start` | The time the last run started |
+| `next_start` | The time the next run will start|
+| `total_runs` | The total number of runs of this job |
+| `total_failures` | The total number of times this job failed |
+
+#### Sample Usage
+
+Get information about current license.
+
+```sql
+SELECT * FROM timescaledb_information.policy_stats;
+       hypertable       | job_id |  job_type   | last_run_success |         last_finish          |          last_start          |          next_start          | total_runs | total_failures 
+------------------------+--------+-------------+------------------+------------------------------+------------------------------+------------------------------+------------+----------------
+ conditions             |   1001 | drop_chunks | t                | Fri Dec 31 16:00:01 1999 PST | Fri Dec 31 16:00:01 1999 PST | Fri Dec 31 16:00:02 1999 PST |          2 |              0
+(1 row)
+```
+
+---
+## timescaledb_information.policy_stats[](timescaledb_information-policy_stats)
```

Comment:
This whole section repeats previous above (cut-and-paste error?) and should be removed?

Reply:
yeah. there's like 2 more views, I was in process on that, but published to begin getting feedback, sorry if it wasn't clear.